docs/website/root/manual/develop/nodes/mithril-signer.md

```diff
@@ -61,7 +61,7 @@ Switch to the desired branch/tag:
 
 ```bash
 # Replace **YOUR_BUILD_BRANCH_OR_TAG** with the appropriate branch or tag name
-# Please refer to the **Build from** column of the **Mithril networks** table above
+# Please refer to the **Build from** column of the **Mithril networks** section
```

Comment:
There are many mentions of this note throughout the guide. I changed the 'table' to 'section'. However, I think it's best to update these notes to say 'Network configurations' and link to a newly created section.

Reply:
Done in https://github.com/input-output-hk/mithril/pull/2102/commits/aada22bb936fdef5740fdcf8a4718e89bfaecc02 :heavy_check_mark: